### DISCUSSION TITLE
Fallout 2 mods

### DISCUSSION THREAD

**Discussion by Lawka89** [Northfear/fallout2-ce-vita#3] [Ideas]
The quality of this port for the Vita is amazing and feels great
Would be even better with additional Fallout 2 content
Is it considered to add support for mods like the "restoration project" in the future?

**Comment by Northfear** [comment]
I think so. Most of the discussions about planned features and issues are found on the page of the original project. Here's the tracker of integration of sfall features for example:
https://github.com/alexbatalov/fallout2-ce/issues/29
I'm just maintaining Vita port here (controls, optimizations, platform specific stuff)
